### ISSUE TITLE
Crush application

### ISSUE THREAD

**Issue by proton2** [m-schuetz/SimLOD#7]
Error log 
[simlod-error-log.txt](https://github.com/m-schuetz/SimLOD/files/14003206/simlod-error-log.txt)

**Comment by proton2** [comment]
Version of cuda:
[version.json](https://github.com/m-schuetz/SimLOD/files/14003197/version.json)

**Comment by m-schuetz** [comment]
Yeah that's probably because of the CUDA version, I guess I typedef'd stuff that's now being defined by newer CUDA versions itself. 

You could try removing the conflicting typedefs in utils.h.cu: https://github.com/m-schuetz/SimLOD/blob/de23f1cb51334726b4718979e461af2db258ffd8/modules/progressive_octree/utils.h.cu#L11-L16

According to the error, int8_t is one of the problems. Maybe there will be another one once you fix that.

**Comment by proton2** [comment]
After this fix I try to run application and I get this error:

<create windows>
<set input callbacks>
<glewInit done> (0.761314)
cpu.numProcessors: 8
launching 16 loader threads
cuMemAlloc(&cptr_buffer,                 300 MB);
cuMemAlloc(&cptr_nodes,                   40 MB);
cuMemAlloc(&cptr_renderbuffer,           200 MB);
cuMemAlloc(&cptr_stats,                  104   );
cuMemAlloc(&cptr_buffer_persistent,    5'841 MB);
==============================================
                                       6'381 MB

================================================================================
building CUDA program
compiling progressive_octree_voxels.cu
CUDA_PATH is set to: F:\apps\Cuda
- compiled in 0.657s
compiling utils.cu
CUDA_PATH is set to: F:\apps\Cuda
- compiled in 0.086s
cuda error code: 200
cuda error code: 200
cuda error code: 200
link error message:
link info message:
cuda error code: 200

SimLOD.exe (process 10816) exited with code -1073741819.
